src/simulation/xml_parser.cpp

```
@@ -283,12 +282,12 @@ void XMLParser::handleBlock(const pugi::xml_node& item,
   std::string name =
       class_name + std::to_string(defined_class_count_[class_name]);
 
-  std::string pos_str = item.child_value("init_pose");
+  std::string       pos_str = item.child_value("init_pose");
   std::stringstream ss(pos_str);
-  std::string axis;
+  std::string       axis;
 
   glm::vec3 pos;
```

Clang (14.0.6)

[error] variable 'pos' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)